README.md

```diff
@@ -166,6 +166,13 @@ brew install --cask spotube
       <p><code>winget install --id KRTirtho.Spotube</code></p>
     </td>
   </tr>
+    <tr>
+        <td>IOS - <a href="https://github.com/KRTirtho/spotube/releases/download/nightly/Spotube-iOS.ipa">IOS</a>
+                </td>
+                <td>
+                    <p><code>IOS Install Spotube</code></p>
```

Comment:
Is `IOS` a command? I don't know this software.
This column is dedicated to command to use to install spotube on every OS.

Reply:
Yes, IOS is hyperliked with that link.

Reply:
> Yes, IOS is hyperliked with that link.

No, that's the iOS on line 170. Its usage here on line 173 is invalid. Please change it.